static/webmention.js

```diff
@@ -54,14 +54,25 @@ GitHub repo (for latest released versions, issue tracking, etc.):
         'follow-of': '🐜',
     };
 
+    var rsvpEmoji = {
+        'yes': '✅',
+        'no': '❌',
+        'interested': '💡'
+        'maybe': '⚖️'
+    };
+
     function reactImage(r) {
         var who = (r.author && r.author.name ? r.author.name : r.url.split('/')[2]);
         var response = reactTitle[r['wm-property']] || 'reacted';
         var html = '<a class="reaction" rel="nofollow" title="' + who + ' ' + response + '" href="' + r.url + '">';
         if (r.author && r.author.photo) {
             html += '<img src="' + r.author.photo + '">';
         }
-        html += (reactEmoji[r['wm-property']] || '⁉️') + '</a>';
+        html += (reactEmoji[r['wm-property']] || '💥');
+        if (r.rsvp && rsvpEmoji[r.rsvp]) {
```

Comment:
It looks like RSVPs can contain a content body, if the person sending the RSVP wants to - may be worth supporting too?

Reply:
By content body, do you mean the text part that's contained within the `p-rsvp` text? In my template implementation I do `<a href="..." class="p-rsvp u-in-reply-to">yes</a>` although the [spec](http://microformats.org/wiki/h-entry#p-rsvp) recommends doing `<data class="p-rsvp" value="yes">text text text</data>` instead, but doesn't specify what should be done with the body of the `<data>` as far as I can tell.

On the one RSVP webmention on the test data, the RSVP itself is formatted as:

```html
I <data class="p-rsvp" value="yes">will be there</data>.</p>
```

but I don't see anything in the JSON blob from the API that just contains "will be there," so if there's an intended purpose to this stuff I don't think webmention.io supports it.

(also, guess I'm gonna fix my RSVP  Publ template)

Reply:
So looking at [the webmentions for https://www.jvt.me/posts/2019/03/14/homebrew-website-club-nottingham/](https://webmention.io/api/mentions.jf2?per-page=100&target[]=http%3A%2F%2Fwww.jvt.me%2Fposts%2F2019%2F03%2F14%2Fhomebrew-website-club-nottingham%2F&target[]=https%3A%2F%2Fwww.jvt.me%2Fposts%2F2019%2F03%2F14%2Fhomebrew-website-club-nottingham%2F) there is the following WebMention with `content`:

```json
    {
      "type": "entry",
      "author": {
        "type": "card",
        "name": "Joshua Hawxwell",
        "photo": "",
        "url": "https://hawx.me/"
      },
      "url": "https://hawx.me/weblog/information/1e85a7bf-e453-41c5-804e-3739a3259aa3/",
      "published": "2019-03-16T16:15:00+00:00",
      "wm-received": "2019-06-02T12:57:03Z",
      "wm-id": 620663,
      "wm-source": "https://hawx.me/weblog/information/1e85a7bf-e453-41c5-804e-3739a3259aa3/",
      "wm-target": "https://www.jvt.me/posts/2019/03/14/homebrew-website-club-nottingham/",
      "content": {
        "html": "<p>Pretty cool that <a href=\"https://www.jvt.me/\">Jamie</a> is going to be starting a\n<a href=\"https://indieweb.org/Homebrew_Website_Club\">Homebrew Website Club</a> in\n<a class=\"u-in-reply-to\" href=\"https://www.jvt.me/posts/2019/03/14/homebrew-website-club-nottingham/\">Nottingham</a>.\nI will be there.</p>",
        "text": "Pretty cool that Jamie is going to be starting a\nHomebrew Website Club in\nNottingham.\nI will be there."
      },
      "rsvp": "yes",
      "in-reply-to": "https://www.jvt.me/posts/2019/03/14/homebrew-website-club-nottingham/",
      "wm-property": "rsvp",
      "wm-private": false
    },
```

Reply:
I guess I'm not sure what you meant by content, then. Because like, reply content is already supported just fine, since rsvps are a type of in-reply-to.

Reply:
Sorry, ignore me then - I've verified it works when using this branch's script :smile: